### PR TITLE
Navigation edits, adding de novo assemply and image path fixes

### PR DESCRIPTION
**Pull request by vperreau**
Added path for Graces's nanopore de novo assembly workshop to navigation
Fixed the broken image paths in nanopore de novo assembly workshop
Combined skills and tools under one heading in navigation as per Dannys suggestion
Some text edits to genome browser workshop- in progress, not yet finished

### CODE REVIEW

File: docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md [hunk 1]
```diff
@@ -25,46 +26,61 @@ For queries relating to this workshop, contact Melbourne Bioinformatics (bioinfo
...
 
-* Understand the ...
-* Be familiar with ...
-* Know how to ...
+* Understand the how different types of genomics and expression data are represented in Genome browsers
+* know how to import and view your own data in a genome browser
+* Know the main file types used for storing expression data BAM file type and
+* Know how to index and sort BAM files using IGV tools
+* Create, save and share custom views data in Genome browsers
+*
```
Comment (junyk) [inline docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md:62]: extra bullet point?

File: docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md [hunk 2]
```diff
@@ -98,19 +114,90 @@ Little bit of history and context. Why is this important/useful...
 
 -------------------------------
 
-## Section 1: Title of section 1
+## Section 1: Intro to Genome Browsers
 
 In this section you will ...
 
 !!! attention "Important"
     **Please look at the [formatting template](/formatting_template/#formats-to-use) to see examples for formatting your documentation in line with Melbourne Bioinformatics training material e.g Questions and answers, code blocks, images etc.**
```
Comment (junyk) [inline docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md:122]: just noting that this url is invalid `/formatting_template/#formats-to-use`

File: docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md [hunk 3]
```diff
@@ -98,19 +114,90 @@ Little bit of history and context. Why is this important/useful...
...
+
+    For further info on Human Genome version updates I recommend you look at the updates and blog pages on the UCSC genome browser:
+    https://genome.ucsc.edu/goldenPath/newsarch.html#2019
+    http://genome.ucsc.edu/blog/patches/
 
 -------------------------------
-## Section 2: Title of section 2
+## Section 2: UCSC genome Browser
+
+In this section we will become familar with e interface of the UCSC genome browser and explore a small number of tools and datasets available.
```
Comment (junyk) [inline docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md:150]: typo: `familar with e interface`

File: docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md [hunk 4]
```diff
@@ -98,19 +114,90 @@ Little bit of history and context. Why is this important/useful...
...
+
+Weekly maintenance of the browser is at 5-6 pm pacific time = 11am-12pm Melbourne time. To ensure uninterrupted browser services for your research during UCSC server maintenance and power outages, bookmark a mirror site that replicates the UCSC genome browser.
```
Comment (junyk) [inline docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md:152]: Do we need to be more specific whether it's AEST or ADST timezone?

Comment (vperreau) [reply docs/tutorials/Genome_browsers/GenomeBrowsers_Intro.md:152]: I am making more edits to this doc over the next few days for delivery as a workshop on Wednesday, it is not yet in the navigation. It is a work in progress.

**Pull request merged**